starter-kit/app/src/App.js

```diff
@@ -5,6 +5,8 @@ import Home from './components/Home/Home';
 import Posts from './components/Posts/Posts';
 import Tasks from './components/Tasks/Tasks';
 import NotFound from './components/NotFound/NotFound';
+import Footer from './components/Footer/Footer';
+import './template/css/core/style.css';
```

Comment:
It looks like style.css is the only template file used but others have been committed too. Also you should import this style sheet where it's applicable/used so it doesn't override other component styling.